src/logtab.cpp

```diff
@@ -608,7 +620,7 @@ void LogTab::CopyItemDetails(const QModelIndexList& idxList)
                 }
                 else
                 {
-                    copyText += "N/A\t";
+                    copyText += "-\t";
```

Comment:
Two reasons for using "-":
- Consistency: This is mainly used for the calculated Elapsed column.  Other fields in our JSON logs will have "-" set when there is no value.
- Easier to identify the empty fields when you paste it in plain text editor.